source/main/gui/panels/GUI_SimActorStats.cpp

```diff
@@ -36,7 +36,7 @@ void RoR::GUI::SimActorStats::Draw(RoR::GfxActor* actorx)
     // TODO: Localize all! See https://github.com/RigsOfRods/rigs-of-rods/issues/1269
     ImGuiWindowFlags flags = ImGuiWindowFlags_NoInputs | ImGuiWindowFlags_NoCollapse |
         ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoTitleBar;
-    ImGui::SetNextWindowPos(ImVec2(25.f, 150.f));
+    ImGui::SetNextWindowPos(ImVec2(10.f, 160.f));
```

Comment:
TIP: use common 'theme' setting 'screen edge padding' (example: https://github.com/RigsOfRods/rigs-of-rods/blob/4f8658fcd653bda8f0ebea05760fc9adf05a13c7/source/main/gui/panels/GUI_MultiplayerClientList.cpp#L53)